src/test/java/com/training/services/tests/GACreateTest.java

```diff
@@ -71,4 +71,107 @@ public void  testGuestCreation(){
         softAssert.assertThat(responseGACreate.getPayload().getRefreshToken()).isNotEmpty();
         softAssert.assertThat(responseGACreate.getPayload().getUid()).isEqualTo(uid);
     }
+
+    /**
+     * testWrongMailGuestCreation():passing wrong mail in reqest body using POJO
+     * to get response status 422
+     */
+    @Test
+    public void  testWrongMailGuestCreation(){
+        String wrongUid=uid+"@@@api.net";
+        requestBodyCreate.setEmail(wrongUid);
+        Response responseGACreate  =
+                new RestEngine().getResponsePost(map.get("URI")
+                        , headerMap
+                        ,new Gson().toJson(requestBodyCreate)).as(Response.class);
```

Comment:
This code looks to be repeating in all our tests. Lets define a method here in test class which does this, and you just need to call this method then in all your tests